### PR TITLE
DE-97 - system_setup: reduce WSLConfigurationAdvanced size

### PR DESCRIPTION
**Pull request by patrick330602**
This fixes the TODO 1 for the WSL OOBE
1. Remove duplicated fields between `wslconfadvanced` and `wslconfbase`.

### CODE REVIEW

File: subiquity/common/types.py [hunk 1]
```diff
@@ -346,7 +346,7 @@ class WSLConfigurationBase:
 
 
 @attr.s(auto_attribs=True)
-class WSLConfigurationAdvanced:
+class WSLConfigurationAdvanced(WSLConfigurationBase):
```
Comment (didrocks) [inline subiquity/common/types.py:349]: WSLConfigurationBase and WSLConfigurationAdvanced are 2 independents models and controllers.

Where in reconfigure mode, we show both base and advanced screen, each one showing its own model.

Do no heritate from it, WSLConfigurationAdvanced are additional properties that are not in WSLConfigurationBase, or said differently, being in setup or reconfigure mode, custom_path, custom_mount_opt, gen_host and gen_revolvconf are always set in WSLConfigurationBase.

File: system_setup/models/wslconfadvanced.py [hunk 2]
```diff
@@ -18,14 +18,15 @@
...
+from .wslconfbase import WSLConfigurationBase
 
 log = logging.getLogger('subiquity.models.wsl_configuration_advanced')
 
 # TODO WSL: Remove all common attributes with wslconfbase
 
 
 @attr.s
-class WSLConfigurationAdvanced(object):
+class WSLConfigurationAdvanced(WSLConfigurationBase):
```
Comment (didrocks) [inline system_setup/models/wslconfadvanced.py:29]: ditto, do not inherite here

**Review by didrocks — CHANGES_REQUESTED** [review]

File: subiquity/common/types.py (hunk 1 above)
Comment (patrick330602) [reply subiquity/common/types.py]: Both are fixed

File: system_setup/models/wslconfadvanced.py (hunk 2 above)
Comment (patrick330602) [reply system_setup/models/wslconfadvanced.py]: Both are fixed

**Comment by didrocks** [comment]
This LGTM, thanks! I’ll let upstream review now.

**Review by didrocks — APPROVED** [review]

**Review by dbungert — APPROVED** [review]

**Comment by dbungert** [comment]
I took the liberty of deleting the branch, if it is still needed we can restore it easily.

**Pull request merged**